frontend/src/lib/stores.ts

```diff
@@ -0,0 +1,7 @@
+import { writable } from "svelte/store";
+
+export const displaySymbol = writable("AAPL");
```

Comment:
As a general point, it's better to define an inline type that encapsulates all of them, e.g. `const currentStock: writable<{ticker: string, name: string, exchange: string}> = writable({ticker: "AAPL", name: "Apple Inc", exchange: "NASDAQ"})>`